vaadin-platform-test/src/test/java/com/vaadin/platform/test/ComponentsIT.java

```diff
@@ -26,7 +26,7 @@ public class ComponentsIT extends AbstractPlatformTest {
         String browsers = System.getProperty("grid.browsers");
         if (sauceUser != null && !sauceUser.isEmpty()) {
             if (browsers == null || browsers.isEmpty()) {
-                Parameters.setGridBrowsers("firefox,chrome,safari,edge");
+                Parameters.setGridBrowsers("firefox,safari,edge");
```

Comment:
curious: why remove chrome from test?

Reply:
Because we run twice this class, this list is only when running in sauce, when the class is run in local or in hub it's always chrome